fediplay.py

```diff
@@ -53,9 +53,12 @@ def get(self, url):
             'nocheckcertificate': 'FEDIPLAY_NO_CHECK_CERTIFICATE' in environ,
             'progress_hooks': [self._progress_hook]
         }
-        with YoutubeDL(options) as downloader:
-            downloader.download([url])
-
+        try:
+            with YoutubeDL(options) as downloader:
+                downloader.download([url])
+        except Exception:
+            print("YoutubeDL couldn't download "+url+" for you. That might not be a valid link or you might be having connection issues.")
+            self.filename=""
```

Comment:
This is too deep to handle the exception—setting `self.filename` to an empty string (essentially a magic value) is evidence of this.

I'd rather see the exception bubble up and be handled at the `StreamListener.on_update` level, and then fix #3 by checking all URLs until youtube-dl successfully handles one.